superset/assets/src/visualizations/core/models/ChartProps.js

```diff
@@ -29,3 +30,44 @@ export default class ChartProps {
     this.setTooltip = setTooltip;
   }
 }
+
+ChartProps.createSelector = function () {
```

Comment:
In some repos I've seen long selectors defined in separate files in a `selectors/` folder. cool with whatever here, if we end up with a lot of selectors we could update to that later.

Reply:
I have seen it more commonly for using with containers

```
- selectors
- containers
- components
```

Since there is only one selector so far let's keep it here. I may revise this later if we push the `ChartProps` construction up to container layer. Nice thing about this one is if we change `ChartProps` fields we have to update only this file.